docs/shared/revisions.txt

```diff
@@ -26,6 +26,7 @@
 [cols="2",options="header"]
 |===
 | Version    | Date
+| 2.1.0      | TBD
 | 2.0.1      | July 7, 2016
 | 2.0.0      | June 6, 2016
 | 1.3.0      | January, 2016  
```

Comment:
Yes, that sounds good. Just add revisions.txt to the list of those files that needs to be updated and docs published. Thanks.